src/core/std_type.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#ifndef STD_TYPE_H
+#define STD_TYPE_H
+
+#include <vector>
+#include <map>
+
+template <class Key, class Value>
+using Map = std::map<Key, Value>;
```

Comment:
not a fan of this file, or its contents
aliasing `std::map` to `Map` is just pointless obfuscation to save 5 characters, and `GetOrDefault` doesn't actually seem to be used anywhere

Reply:
I like simple and clear names. I believe that in this case it is obvious that Map is a key-value store. And the details of the implementation should not be important.
In addition, I do not force anyone to connect and use the library. I have big plans, and I want to have a handy toolkit that will allow me to focus on the idea, and not on the task of finding an item, storing lists, keys, values, and so on.

Reply:
It goes against the coding conventions in the rest of the project, and in fact we have recently moved away from using custom types, to using STL types directly. Please remove this file.